fmt/printf.ts

```diff
@@ -450,7 +458,9 @@ class Printf {
     // deno-lint-ignore no-explicit-any
     const arg = this.args[this.argNum] as any;
     if ((arg || {}).constructor.name !== "Array") {
-      throw new Error(`arg ${arg} is not an array. Todo better error handling`);
+      throw new Error(
+        `Printf must be constructed with an array for 'arg': got ${arg}`,
```

Comment:
Because of the below rules:

> Message should state the action that lead to the error:
> Bad: Invalid input x
> Good: Cannot parse input x

> Messages should use a colon when providing additional information:
Bad: Cannot parse input x. value is empty
Bad: Cannot parse input x, value is empty
Good: Cannot parse input x: value is empty

I think the message should be something like the below:

```suggestion
        `Cannot handle less than '<' flag: 'arg' is not an array`,
```